Entities/ACrab.cpp

```diff
@@ -59,6 +59,7 @@ void ACrab::Clear()
     m_pJetpack = 0;
     m_JetTimeTotal = 0.0;
     m_JetTimeLeft = 0.0;
+	m_JetReplenishRate = 1.75F;
```

Comment:
I'm still on the fence about this, maybe the property could be named better so that it clearly explains that it's for the replenish rate in relation to the depletion rate, I think only that way it makes sense. Or maybe the way you suggest, I guess it's fine.

Reply:
Yeah, I get what you're saying. If you can think of a name to clearly convey that, I guess that'd be okay too. Right now I think it's confusing, for the reasons I've mentioned before.